src/style.css

```diff
@@ -0,0 +1,133 @@
+.page {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 300;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  width: 198px;
+  height: 406px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px;
+  display: flex;
+  justify-content: center;
+}
+
+.card__content {
+  margin: 0 16px 5px;
+  display: flex;
+  flex-wrap: 1;
+  font-weight: 500;
+  font-size: 12px;
+  color: #060b35;
+  line-height: 18px;
+}
+
+.card__prescription {
+  margin: 0 16px 18px;
+  font-weight: 400;
+  font-size: 10px;
+  color: #616070;
+}
+
+.rating {
+  margin: 0 16px 23px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars-item {
+  background-image: url(./images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars--4 > .stars-item:nth-child(-n + 4) {
+  background-image: url(./images/star-active.svg);
+}
```

Comment:
All modifiers should work.
If I change .stars--4 class to .stars--2, should be 2 active stars
<img width="533" alt="Screenshot 2022-07-20 at 10 04 29" src="https://user-images.githubusercontent.com/47606379/179918310-fcb04ec5-4fd2-400d-bcb8-74b8e96fdc4a.png">